seed/static/seed/partials/organization_settings.html

```diff
@@ -217,7 +217,7 @@ <h3 translate>THERMAL_CONV_ASSUMPTION_TITLE</h3>
             </div>
         </uib-tab>
 
-        <uib-tab index="3" heading="{$:: 'Default Display Fields' | translate $}">
```

Comment:
The changes made in this file are no longer necessary and should be reverted back